bundle/src/main/java/com/cognifide/cq/cqsm/api/history/Entry.java

```diff
@@ -135,4 +132,22 @@ public int hashCode() {
 	public boolean equals(Object obj) {
 		return EqualsBuilder.reflectionEquals(this, obj);
 	}
-}
+
+	public String getExecutionResultFileName() {
+		return EXECUTION + "-" + StringUtils.replace(fileName, ".cqsm", ".txt");

```

Comment:
how about two types of file extensions to be supported?

My dream is to leave 'cqsm' ...at all